Civi/Api4/Action/Queue/RunItems.php

```diff
@@ -56,7 +56,7 @@ public function _run(\Civi\Api4\Generic\Result $result) {
     if (!empty($this->items)) {
       $this->validateItemStubs();
       $queue = \Civi::queue($this->items[0]['queue']);
-      $ids = \CRM_Utils_Array::collect('id', $this->items);
+      $ids = array_column($this->items, 'id');
```

Comment:
in this case `$ids` is passed to `fetchItems` but it doesn't seem like the key matters

Reply:
Right.